recipes/fgsl/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
maybe remoev the comments here?

Reply:
Addressed via https://github.com/conda-forge/staged-recipes/pull/16107/commits/962c83fb25a068029f3e64e6794de862fe35a461